airsonic-main/src/main/java/org/airsonic/player/service/SettingsService.java

```diff
@@ -723,6 +731,30 @@ public void setGettingStartedEnabled(boolean isGettingStartedEnabled) {
         setBoolean(KEY_GETTING_STARTED_ENABLED, isGettingStartedEnabled);
     }
 
+    public boolean isExternalServicesChromecastEnabled() {
```

Comment:
I'm quite new to Java, so maybe it's idiomatic, but couldn't we add only one method to check for these settings, passing the name of the service as the first argument? That would allow for less code duplication, no?

Reply:
Yes that would allow for less code duplication, however in this case the duplication isn't terrible as it keeps everything consistent across the whole class. By keeping things consistent it makes it easier to address the code duplication at a larger scale in the future (i.e the whole class).

Reply:
I understand! (Is there an already open issue to fix this code duplication at some point, or should we open one?)

About this issue, should we move ahead and merge this? If not, what's missing?

Reply:
I don't think there is an open issue for it, but its pretty low priority in my mind. So low I wouldn't even bother opening an issue for it.

This PR is still marked as WIP which means the author doesn't think its ready to get merged yet.